README.md

```diff
@@ -1,21 +1,112 @@
+
+## Releasing a new version
+
+You will need a 64-bit Windows PC. To release the version for macOS as well, you will also need a Mac.
+
+1. If the randomizer version for league seeds should be updated, change the constants `LEAGUE_COMMIT_HASH` and `LEAGUE_VERSION` in `crate/ootr/src/lib.rs` accordingly. `LEAGUE_COMMIT_HASH` is the commit hash for the Dev-R version to be used, which can be copied using the clipboard button at <https://github.com/Roman971/OoT-Randomizer/commits/Dev-R>, and `LEAGUE_VERSION` is the contents of the `__version__` variable in `version.py` in that commit.
+2. Bump the version number in:
+    * `assets/macos/RSL.app/Contents/Info.plist` (change `CFBundleShortVersionString` to the new version number and increment `CFBundleVersion` by 1)
+    * `crate/ootr/Cargo.toml`
+    * `crate/rsl/Cargo.toml`
+    * `crate/rsl-cli/Cargo.toml`
+    * `crate/rsl-gui/Cargo.toml`
+    * `crate/rsl-updater/Cargo.toml`
+    * `crate/rsl-utils/Cargo.toml`
+    * `crate/rslbot/Cargo.toml`
+3. Push the changes
+4. On macOS (if releasing for macOS):
+    1. Install [Rust](https://rust-lang.org/) and [Python](https://python.org/).
+    2. Make sure your Rust compiler can build for both Intel and ARM:
+        ```sh
+        rustup target add x86_64-apple-darwin
+        rustup target add aarch64-apple-darwin
+        ```
+    3. Clone this repo.
+    4. In System Preferences → Sharing, make sure Remote Login is enabled and you have access.
+5. On Windows:
+    1. [Create a GitHub personal access token](https://github.com/settings/tokens/new) with the public_repo scope.
+    1. Install [Rust](https://rust-lang.org/) and [Python](https://python.org/).
+    2. Clone this repo.
+    3. In the repo clone, create a [JSON](https://json.org/) object at `assets\release-config.json` with the following entries:
+        * `"githubToken"`: The personal access token from step 1
+        * `"macHostname"`: A hostname or IP address (and if necessary the username, in `"username@hostname"` form) at which your Mac is reachable via SSH (optional, if not given no macOS app is included in the release)
+        * `"macRepoPath"`: The path to the clone of this repo on your Mac (optional, if not given no macOS app is included in the release)
+        * `"releaseNotesEditor"`: A command that will be called with `--wait` and the path to an empty file for writing the release notes (optional, defaults to `"C:\\Program Files\\Microsoft VS Code\\bin\\code.cmd"`)
+        * `"repoOwner"`: The GitHub user or org where the release should be published (optional, defaults to `"matthewkirby"`)
+        * `"repoName"`: The name of the repository where the release should be published (optional, defaults to `"plando-random-settings"`)
+    4. In the repo, run `cargo run --release --package=rsl-utils --bin=rsl-release`. You can add the following command-line options, but must separate them with a `--` (so they're interpreted as `rsl-release` options, not `cargo` options):
```

Comment:
I haven't been able to get a successful release. It's a rustc error apparently because the x86_64-pc-windows-mvc release target is missing.

Here's my [branch](https://github.com/SlyryD/plando-random-settings).

Here's my [release-config.json](https://github.com/matthewkirby/plando-random-settings/files/6041462/release-config.json.txt) (without token)

Here's the [output](https://github.com/matthewkirby/plando-random-settings/files/6041445/cargolog.txt) of the command.

Reply:
That was caused by a typo in the release script. Apparently I forgot to test this after adding the updater. Try again please.

Reply:
Tested a release, seems to be working again.